### DISCUSSION TITLE
can see the client in the gui but can't watch or control it (no window opens)

### DISCUSSION THREAD

**Discussion by jorigel** [epoptes/epoptes#215] [Q&A]
Hi,
the installation is done like discribed. Internal clients work. An external is seen but can't be watched or controlled because no window opens. If gui is called via terminal it says: "watching port 46xxx . . ." but nothing happens at that point.

What is wrong here?
Thank You for Ur attention
jori

**Comment by miguel-6-code** [comment]
Same to me.
Epoptes had works fine on ubuntu 20 but since I've installed ubuntu 22, I can see all the clientes on the gui and even restart, close sesions, block screens.... but I can't watch their scrreens neither control their computers.  
I've unistalled and install again both server and client, but the same.   :-/

do you knows whats is wrong?

thanks in advance !!

**Comment by jorigel** [comment]
Hi miguel,

in my surrounding it works following this:
- opening epoptes on the master mashine via terminal shows the upper mentioned message "watching port soandso"
- now using that port number to open the port on the router to give the client the chance to reach the master through the firewall

now the whole thing works like expected and all functions are ready to use. I know, this is not like we want it to be. We expect it to work out of the box without opening ports for every start of epoptes.

I did not find a final solution for that up to now and I hope that alkis will find some time to give us a hint what to do about that :-)

regards
